ext/DimensionalDataMakie.jl

```diff
@@ -161,6 +161,14 @@ for (f1, f2) in _paired(:plot => :heatmap, :heatmap, :image, :contour, :contourf
             # No ColourBar in the ! in-place versions
             return Makie.$f2!(axis, args...; attributes...)
         end
+        function Makie.$f1!(axis, A::Observable{<:AbstractDimMatrix};
```

Comment:
Maybe also change all the ones above to also use `AbstractDimMatrix` for consistency, maybe it didn't exist back then...

Reply:
I changed them to AbstractDimMatrix and AbstractDimVector in the whole file.